docs/tools/link_filter.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+import panflute as pf  # type: ignore
+
+# Thanks to Sergio Correia. Reference:
+# https://stackoverflow.com/questions/40993488/convert-markdown-links-to-html-with-pandoc
```

Comment:
Is this called by doc_utils? If so, it doesn't need too.

Reply:
Yes, it's called by doc_utils. But if doc_utils is not needed, this is not needed as well.

Reply:
Both of them are not needed, because it's one time work.